dashboard/app/models/census/state_cs_offering.rb

```diff
@@ -36,6 +37,13 @@ def self.construct_state_school_id(state_code, row_hash)
       School.construct_state_school_id('GA', row_hash['SYSTEM_ID'], school_id)
     when 'ID'
       School.construct_state_school_id('ID', row_hash['LeaNumber'], row_hash['SchoolNumber'])
+    when 'NC'
+      # school code in the spreadsheet from North Carolina is prefixed with the district code
```

Comment:
nit: periods at end of sentences please :)

Reply:
Done!